### PR TITLE
ObjectId Sort

### PR DESCRIPTION
**Pull request by Yuqi-Du**
**Checklist**
- [ ] Changes manually tested
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [ ] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

**Review by tatu-at-datastax — COMMENTED** [review]
LGTM. I think @maheshrajamani would be good person to additionally review, just sanity checking.

**Review by tatu-at-datastax — APPROVED** [review]
Approving just to make sure it can be merged; no specific concerns.

File: src/test/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/FindOperationTest.java [hunk 1]
```diff
@@ -2467,6 +2467,132 @@ public void findAllSortDescending() throws Exception {
...
+    String collectionReadCql =
+        "SELECT key, tx_id, doc_json, query_text_values['uuidv6'], query_dbl_values['uuidv6'], query_bool_values['uuidv6'], query_null_values['uuidv6'], query_timestamp_values['uuidv6'] FROM \"%s\".\"%s\" LIMIT %s"
+            .formatted(KEYSPACE_NAME, COLLECTION_NAME, 20);
+
+    // These are UUIDv6 ids generated at 30 second intervals.
+    String doc1 =
+        """
+                      {
+                        "_id": "doc1",
+                        "uuidv6": "1e712685-714f-6720-a23a-c90103f70be6"
```
Comment (vkarpov15) [inline src/test/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/FindOperationTest.java:2482]: What happens if you try to sort documents with mixed types, like they have a property `myId` and some documents have `myId` as a `uuidv6` and some have `myId` as a `uuidv7`?

**Review by vkarpov15 — APPROVED** [review]
One small question, LGTM overall

**Review by maheshrajamani — APPROVED** [review]

File: src/test/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/FindOperationTest.java (hunk 1 above)
Comment (Yuqi-Du) [reply src/test/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/FindOperationTest.java:2482]: So, what we did for sort is just lexicographic by String representation. 
If there is a mix of uuidv6 and uuidv7, still sort in lexicographic order.
If I understand correctly, the timestamp design for v6 and v7 are different, then the result will not necessarily be in time order.

**Pull request merged**